content/millennium/dstu2/general-clinical/condition.md

```diff
@@ -26,64 +53,108 @@ Search for Conditions that meet supplied query parameters:
  `clinicalstatus` | N                                                                    | [`token`]     | The [clinical status] of the condition. Example: `resolved`
 
-Note: Currently `diagnosis`, `problem` and `health-concern` category codes are supported.
+Notes:
+ 
+* Currently only `diagnosis`, `problem` and `health-concern` category codes are supported.
+
+### Headers
+
+ <%= headers %>
 
-### Response
+### Example
 
-<%= headers status: 200, head: {Functionality: 'Search by patient'} %>
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition?patient=4342012
+
+#### Response
+
+<%= headers status: 200 %>
 <%= json(:dstu2_condition_bundle) %>
-<%= headers status: 200, head: {Functionality: 'Search by id'} %>
+
+### Example Read by Ids
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition?_id=d35686553,p6317881,7965c636-f343-4e96-8904-3bc35e9fd9f9
+
+#### Response
+
+<%= headers status: 200 %>
 <%= json(:dstu2_condition_bundle_by_id) %>
 
+### Errors
+
+The common [errors] may be returned.
+
 ## Retrieve by id
 
 List an individual Condition by its id:
 
     GET /Condition/:id
 
-### Response
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true)%>
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
 
-<%= headers status: 200, head: {Functionality: 'Retrieve diagnosis by id'} %>
-<%= json(:dstu2_condition_diagnosis_resource) %>
-<%= headers status: 200, head: {Functionality: 'Retrieve health concern by id'} %>
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition/7965c636-f343-4e96-8904-3bc35e9fd9f9
+
+#### Response
+
+<%= headers status: 200 %>
 <%= json(:dstu2_condition_health_concern_resource) %>
-<%= headers status: 200, head: {Functionality: 'Retrieve problem by id'} %>
-<%= json(:dstu2_condition_problem_resource) %>
+
+### Errors
+
+The common [errors] may be returned.
 
 ## Create
 
 Create a new condition.
 
     POST /Condition
 
-### Headers
+_Implementation Notes_
 
-To successfully POST a condition, the following headers must be provided. Condition creation is supported from the closed endpoint only.
+* Currently, `health-concern` category code is not supported for writing conditions.
 
-    Content-Type: application/json+fhir
-    Authorization: <OAuth2 Bearer Token>
+### Authorization Types
 
-_Implementation Notes_
+<%= authorization_types(practitioner: true, system: true)%>
 
-* Currently, `health-concern` category code is not supported for writing conditions.
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
 
 ### Body fields
 
+Notes:
+
+* dateRecorded is currently not honored on a create for Conditions with a category of `problem`.
```

Comment:
Can you clarify? I'm not sure I'm seeing what you're seeing/may be missing something.